.storybook/preview.js

```diff
@@ -14,7 +14,16 @@ export const parameters = {
   options: {
     storySort: (a, b) => a[1].kind === b[1].kind ? 0 : a[1].id.localeCompare(b[1].id, undefined, { numeric: true }),
   },
-  a11y: {},
+  a11y: {
+    config: {
+      rules: [
+        {
+          id: 'aria-hidden-focus',
+          selector: 'body main *:not([data-rsp-a11y-ignore])',
```

Comment:
Will this miss a HiddenSelect rendered within a Dialog that appears in a Portal at the end of the DOM after the `main`?

```suggestion
          selector: 'body *:not([data-rsp-a11y-ignore])',
```

Reply:
Ah true, good catch.